src/app/Main/Header/Export/Header-ExportContainer.component.tsx

```diff
@@ -88,13 +102,10 @@ export function HeaderExportContainerComponent(): JSX.Element {
     };
 
     useEffect(() => {
-        fetchAndChooseData(setting.raw, setting.stat);
-    }, [
-        table.statisticDataAbsolute,
-        setting.raw,
-        setting.stat,
-        isolateFilteredUrl,
-    ]);
+        if (isExport) {
+            fetchAndChooseData(setting.raw, setting.stat);
+        }
+    }, [isExport]);

```

Comment:
I think the effect and export state is not necessary here, you can just use:
```
handleExport = () => {
  setIsOpen(false);
  fetchAndChooseData(...);
}
```

Reply:
The async export needs to be inside an Effect.